real_world: true

figure_finder:
  figure_collector:
    group_num_instances: 7
    group_dist_thresh: 1.5
    use_local_coords: true
  figure_operations_config_file: config/figure_operations.yaml  
  color_detection_config_file: config/color_detection.json 

detection:
  nn_model_path: nn_models/best_barrel_unity.pt
  detector: yolo

tf_start_pose:
  drone_z_offset: -0.2

#additional settings
upload_data: true